src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -2347,43 +2353,69 @@ private IEnumerable<PSModuleInfo> CreateFakeModuleObject(IEnumerable<ModuleSpeci
             bool isConsideredCompatible = ModuleUtils.IsPSEditionCompatible(moduleManifestPath, inferredCompatiblePSEditions);
             if (!BaseSkipEditionCheck && !isConsideredCompatible)
             {
-                containedErrors = true;
-
-                if (writingErrors)
+                if (ExperimentalFeature.IsEnabled("PSWinCompat"))
+                {
+                    if (importingModule)
+                    {
+                        var commandInfo = new CmdletInfo("Import-Module", typeof(ImportModuleCommand));
+                        var ps = System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace)
+                        .AddCommand(commandInfo)
+                        .AddParameter("PassThru", true)
+                        .AddParameter("Name", moduleManifestPath)
+                        .AddParameter("UseWindowsPowerShell", true);
+                        
+                        var moduleProxies = ps.Invoke<PSModuleInfo>();
+                        // we are loading by a single ManifestPath so expect max of 1
```

Comment:
Please add new line.
```suggestion

                        // we are loading by a single ManifestPath so expect max of 1
```

Reply:
Added.